content/en/blog/_posts/2020-09-02-scaling-kubernetes-networking-endpointslices.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog
+title: 'Scaling Kubernetes Networking With EndpointSlices'
+date: 2020-09-02
+slug: scaling-kubernetes-networking-with-endpointSlices
+---
+
+**Author:** Rob Scott (Google)
+
+EndpointSlices are an exciting new API that provides a scalable and extensible alternative to the Endpoints API. EndpointSlices track IP addresses, ports, readiness, and topology information for Pods backing a Service.
+
+In Kubernetes 1.19 this feature will be enabled by default with kube-proxy reading from [EndpointSlices](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) instead of Endpoints. Although this will mostly be an invisible change, it should result in noticeable scalability improvements in large clusters. It will also enable significant new features in future Kubernetes releases like Topology Aware Routing.
```

Comment:
1. Kubernetes v1.19 is already released, so I recommend tweaking the wording.
2. Better to link to `/docs/concepts/services-networking/endpoint-slices/`
3. EndpointSlice is naturally singular whereas Endpoints is naturally plural, so I'd change the link text to EndpointSlice and add “objects”.

Reply:
1. Does "is" or "has been" work here?
2. Good catch, thanks!
3. To clarify, should all plural EndpointSlice references be stated as "EndpointSlice resources"? Is this just an issue on line 12 because EndpointSlices are being contrasted with Endpoints? I'd thought I might be able to get away with it here since the plural of both names is used but admit it could be confusing.